src/gt4py/backend/gtc_backend/defir_to_gtir.py

```diff
@@ -132,13 +133,28 @@ def visit_StencilDefinition(self, node: StencilDefinition) -> gtir.Stencil:
         field_params = {f.name: self.visit(f) for f in node.api_fields}
         scalar_params = {p.name: self.visit(p) for p in node.parameters}
         vertical_loops = [self.visit(c) for c in node.computations if c.body.stmts]
+        if node.externals is not None:
+            externals = {name: str(value) for name, value in node.externals.items()}
```

Comment:
```suggestion
            externals = {name: repr(value) for name, value in node.externals.items()}
```
Not completely sure, but at least this was the previous strategy used in `generate_constants`.

Reply:
Solved a different way by using `gtir.Literal` instead of strings.